methods.tex

```diff
@@ -349,7 +349,7 @@ \subsection*{Random number generation}
 wraps the core pseudorandom number generator based on a Mersenne Twister
 implementation\footnote{to be precise, the standard 32-bit version of MT19937};
 interfaces with the underlying code that transforms random bits into
-deviates from other distributions; and supplies a singleton instance exposed in
+variates from other distributions; and supplies a singleton instance exposed in
```

Comment:
I stumbled over "deviates" a bit, probably just me, but variates was used all over.

Reply:
deviates should probably used throughout; not sure how that slipped in there!

Reply:
I think "variates" is correct, no?

> A random variate is a particular outcome of a random variable: the random variates which are other outcomes of the same random variable might have different values. A random deviate or simply deviate is the difference of random variate with respect to the distribution central location (e.g., mean), often divided by the standard deviation of the distribution.

- https://en.wikipedia.org/wiki/Random_variate